app/views/orders/_cart_row.html.haml

```diff
@@ -36,8 +26,3 @@
     - if @order.has_subsidies?
       %td.currency Unassigned
```

Comment:
"Unassigned" probably shows up often enough in different spots in the app to be worth having a `shared` translation.